app/display/editor/src/main/resources/display_editor_preferences.properties

```diff
@@ -16,3 +16,6 @@ hidden_widget_types=linear-meter,knob,gauge,clock,digital_clock
 #
 # GUI Menu action Applications / Display / New Display opens the following template
 new_display_template=examples:/initial.bob
+
+# Size of undo stack. Defaults to 50 if not set.
+# undo_stack.size=
```

Comment:
All the other preferences use this pattern to describe the value and define it with its default value:
```
# Description
the_variable=default_value
```

So I'd make this
```
# Size of undo stack
undo_stack_size=50
```

https://control-system-studio.readthedocs.io/en/latest/preferences.html
https://control-system-studio.readthedocs.io/en/latest/preference_properties.html